dataprofiler/profilers/profile_builder.py

```diff
@@ -683,3 +687,127 @@ def tqdm(l):
             
         return profile
 
+    def _delete_data_labelers(self):
+        """
+        Helper method for deleting all data labelers before saving to disk.
+        """
+        # Delete data labeler for profiler
+        data_labeler_options = self.options.structured_options.data_labeler
+        if data_labeler_options.is_enabled \
+                and data_labeler_options.data_labeler_object is not None:
+                    data_labeler_options.data_labeler_object = None
+                
+        # Delete data labelers for all columns
+        for key in self._profile:
+
+            val = self._profile[key]
+            use_data_labeler = True
+            if val.options and isinstance(val.options, StructuredOptions):
+                use_data_labeler = val.options.data_labeler.is_enabled
+
+            if use_data_labeler:
+                val.profiles['data_label_profile']._profiles['data_labeler'] \
+                   .data_labeler = None
+
+    def _restore_data_labelers(self):
```

Comment:
@lettergram can correctly if i'm wrong, but for now, we assume that each column would have the same DL. Hence, if we need the DL to be loaded, we should load it just once.

Reply:
We should load data labeler once for all columns. IMO we should rename this function to load_data_labelers or something, as we probably don't need the _delete_data_labeler function as per my above comment.

Reply:
restore seems reasonable if it is just swapping them and not reloading the actual DL.

Reply:
@AnhTruong @JGSweets do all columns share the same Data Labeler? I wasn't sure which is why I took two steps here - one for the profile, one for all the columns. Because I am setting the pointer to None, I would still need to do this (set all pointers to None), but if I use `del`, I could probably circumvent this. I would only want to make this assumption if all columns and the profiler always share the same Data Labeler object (otherwise the save would break when pickling if multiple data labelers were used)